### ISSUE TITLE
Custom `displayname` breaks `khal` commands

### ISSUE THREAD

**Issue by amandabee** [pimutils/khal#783]
I'm using khal, version 0.9.8.  

If I do `khal new -i` I cannot use a calendar that has a custom display name. I have two calendars with a custom "displayname" file ("Home" and "Classes"). When I use the `?` to see the list of calendars both appear with a blank line after them and khal accepts neither the display name nor the default name for "calendar"

```
$ khal new -i
calendar: ?
Home 

birthdays
contact_birthdays
Classes

logistics
ucb_abh
us_holidays
calendar: Home
unknown calendar enter ? for list
calendar: "Home"
unknown calendar enter ? for list
calendar: household_shared 
unknown calendar enter ? for list
calendar: 
```

If I remove the "displayname" file, I can use the default name to add items just fine. Same thing happens with other `khal` commands, eg.  `khal calendar -a Home` or `khal list -a household_shared` both get me an "Unknown calendar" error:

> Error: Invalid value for "--include-calendar" / "-a": Unknown calendar Home, run `khal printcalendars` to get a list of all configured calendars.

There's no line break in the `displayname` file. Is this a bug or am I doing something super wrong here?

**Comment by amandabee** [comment]
I suspect this is related to #765

**Comment by geier** [comment]
Thank you for reporting this error, I'll have a look ASAP.

**Comment by f-dj** [comment]
How is the status? I just ran into this error and was already suspecting that the displayname is breaking it.

I use fruux and my calendar names are all over the place. Like 20+ character decimal strings. `displayname` made todoman and khal usable again, but this bug hits really hard when importing .ics files with many vevents included. Would be very happy to have this resolved!

**Comment by d7415** [comment]
This should be fixed in #979. I'm holding off merging in case I've missed any unintended consequences.
